### DISCUSSION TITLE
1.22.2

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#10473] [General]
# Notes
- A new firmware for adapters based on the CC1352 and CC2652 has been released, check the [release](https://github.com/Koenkk/Z-Stack-firmware/releases/tag/Z-Stack_Home_3.x.0_20211217) for the files and changelog.
- For those using external converters for TuYa devices: the names of the commands and payloads have been changed for TuYa specific commands. Your external converter probably has to be updated, [this changeset](https://github.com/Koenkk/zigbee-herdsman-converters/commit/27b4ef37377a1f3abd4071a882c47400810f0d8f) should help you on what to change.

# Improvements
- [#3452](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3452) Setup battery % reporting for Silvercrest HG06336
- #10045 Support `power_on_behavior` and `backlight_mode` for TS011F_2_gang_wall
- [#3496](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3496) Support battery % reporting for TuYa TS0043
- [#3491](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3491) Increase range and precision for exposed local temperature calibration
- [#3526](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3526) Add `radiator_covered` for Danfoss 014G2461
- [#3499](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3499) Add `programming_operation_mode` for Popp 701721, Hive UK7004240 and Danfoss 014G2461 
- [#3532](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3532) Improvements for TV02-Zigbee and BRT-100-TRV
- [#3545](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3545) Support electrical measurements for Sinope TH1300ZB
- #10259 Improve battery % accuracy for SNZB-01 and SNZB-03
- [#3558](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3558) Support OTA for Xiaomi QBCZ15LM 
- [#3557](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3557) Expose `action_duration` for Philips 324131092621
- [#3568](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3568) Expose `power_on_behavior` for IKEA TRADFRI drivers
- Show coordinator ieee address in frontend (settings -> about)
- [#3499](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3499) Expose `programming_operation_mode` for Danfoss 014G2461, Hive UK7004240 and Popp 701721
- Allow read/write to unsupported devices
- #6408 Allow to enable white channel of TuYa TS0504B
- #10379 Poll TuYa devices for updates when bound remote changes its state
- [#3585](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3585) Expose temperature for Xiaomi MCCGQ14LM
- [#3586](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3586) Fix TuYa MIR-HE200-TY `motion_direction` exposes
- #10414 Expose power measurements for SmartThings IM6001-OTP05
- [#3596](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3596) Expose heating for SEA801-Zigbee/SEA802-Zigbee

# Fixes
- [#3445](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3445) Fix max `boost_time` for TuYa TS0601_thermostat 
- #9939 Fix Home Assistant template error for update sensor
- #8934 Fix Lidl Silvercrest thermostat 368308_2010 making Home Assistant addon crash
- #7423 Fix `last_seen` not always updated
- #9972 Fix Home Assistant "Invalid value for" warning for number entities
- [#3464](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3464) Fix Neo NAS-PD07 humidity/temperature not always correct
- [#3264](https://github.com/Koenkk/zigbee-herdsman-converters/issues/3264) Fix `msg.device.getEndpoint is not a function` exception  for HY08WE
- #9952 Fix battery/voltage being 0/null for Aqara Opple devices
- #9919 Fix HEIMAN HS3SA detected as HS1SA
- Allow to disable measurement polling for TS0121_plug, HS2SK_nxp and TS011F_plug_3
- #9962 Validate QOS value
- #7716 #7150 Fix IKEA remotes not working after OTA update version >= 2.3.75
- [#3483](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3483) Fix SONOFF SNZB-02 temperature reporting interval
- #9797 Fix power measurements not working for some TS011F plugs
- #10099 Fix filtered attributes being published as `null`
- #7272 Fix energy not reported for TS0121_plug
- [#3505](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3505) Disable color temperature for Philips 433714
- #8287 Fix error for Home Assistant last_seen sensor when epoch is used
- #9907 Cleanup availability experimental log
- [#472](https://github.com/Koenkk/zigbee-herdsman/issues/472) Fix `Cluster 'XXX' has no attribute 'XXX'` error on read response
- #10108 Fix device data persisted when device is removed
- [#3515](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3515) Fix Xiaomi QBKG19LM, QBKG20LM, QBKG39LM, QBKG41LM and QBCZ15LM `state` in payload without endpoint
- #10174 Fix crash with error: `Error: Cluster with name 'XXX' does not exist`
- [#3520](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3520) Fix Sinope TH1124ZB and TH1300ZB `pi_heating_demand` value
- [#3498](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3498) TuYa MIR-HE200-TY make `presence` and `motion` bool instead of string
- #10120 Fix list not overridable in Home Assistant discovery payload
- [#3539](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3539) Fix configure reporting error for TuYa TS0043
- #10131 Fix device properties (e.g `swBuildId`) not being updated on report by device
- [#3351](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3551) Fix Sinope TH1123ZB configuration
- [#3553](https://github.com/Koenkk/zigbee-herdsman-converters/issues/3553) Fix Xiaomi QBKG25LM and QBKG26LM actions
- #10008 Fix double actions for IKEA E2001/E2002 and E1812
- [#3567](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3567) Fix running state not reported for Acova PERCALE2
- [#3574](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3574) Fix Xiaomi MCCGQ12LM not working
- [#3576](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3576) Fix max `load_room_mean ` value for Danfoss 014G2461, Hive UK7004240 and Popp 701721
- #10355 Fix TOGGLE not working when transition is used
- [#3584](https://github.com/Koenkk/zigbee-herdsman-converters/pull/3583) Fix Sinope TH1300ZB configure
- [#10403](https://github.com/Koenkk/zigbee2mqtt/issues/10403) Fix AC26940/AC31266 not reporting state when controlled physically
- #9646 Fix physical state changes not reported for Schneider Electric 545D6514
- #10423 Fix set message with endpoint in payload where the endpoint name ends with a number not working

# New supported devices
This release adds support for **55** new devices:
- `14153905L` Lidl Livarno Home LED floor lamp
- `1743430P7` Philips Hue Impress outdoor Pedestal
- `1743730P7` Philips Hue Calla outdoor
- `1746447P7` Philips Hue outdoor Resonate wall lamp (silver)
- `3802961` Namron LED 9W CCT E27
- `4058075168572` LEDVANCE SMART+ lamp E27 turntable white
- `4062172044776_2` OSRAM Zigbee 3.0 DALI CONV LI dimmer for DALI-based luminaires (one device and pushbutton)
- `4062172044776_3` OSRAM Zigbee 3.0 DALI CONV LI dimmer for DALI-based luminaires (with two devices)
- `4062172044776_4` OSRAM Zigbee 3.0 DALI CONV LI dimmer for DALI-based luminaires (with two devices and pushbutton)
- `5128.10` Iluminize Zigbee 3.0 switch shutter SW with level control
- `75564` Sylvania Smart+ adjustable white and full color bulb A19
- `8718801528334` Ynoa Remote control one button dimmer
- `915005986901` Philips Hue White and color ambiance Gradient Signe table lamp (white)
- `929003054001` Philips Hue Wellness table lamp
- `9290030674` Philips Hue motion outdoor sensor
- `929003099001` Philips Hue white ambiance Aurelle square panel light
- `99140-031` Kwikset SmartCode traditional electronic deadbolt
- `CCT5015-0001` Schneider Electric Roller shutter module
- `E6` Nous Temperature & humidity LCD sensor
- `EFEKTA_PWS_Max` Custom devices (DiY) [Plant watering sensor EFEKTA PWS max](http://efektalab.com/PWS_Max)
- `EFEKTA_THP` Custom devices (DiY) [DIY temperature, humidity and atmospheric pressure sensor, long battery life](http://efektalab.com/eON_THP)
- `EFEKTA_eON213wz` Custom devices (DiY) [Mini weather station, digital barometer, forecast, charts, temperature, humidity](http://efektalab.com/eON213wz)
- `EFEKTA_eON213z` Custom devices (DiY) [Temperature and humidity sensor with e-ink2.13](http://efektalab.com/eON213z)
- `EFEKTA_miniPWS` Custom devices (DiY) [Mini plant wattering sensor](http://efektalab.com/miniPWS)
- `EKO09716` Schneider Electric Zigbee smart socket with power meter
- `EKO09738` Schneider Electric Zigbee smart socket with power meter
- `ERS-10TZBVK-AA` TuYa Smart knob
- `GL-SD-002` Gledopto Zigbee 3.0 smart home switch
- `HAL500` Schwaiger LED bulb GU10 350 lumen, dimmable, color, white 2700-6500K
- `HG06620` Lidl Silvercrest garden spike with 2 sockets
- `KB-B540R-ZB` Dawon DNS IOT smart plug 16A
- `KB-HD100-ZB` Dawon DNS IOT Card holder
- `MCCGQ13LM` Xiaomi Aqara P1 door & window contact sensor
- `NUE-AUWZO2` Nue / 3A Smart Zigbee double power point
- `PERCALE2` Acova Percale 2 heater
- `PRL-1ZBS-12/24V` Climax Zigbee 12-24V relay controller
- `PS600` Salus Controls Pipe temperature sensor
- `ROB_200-016-0` ROBB smart RGB CCT DIM 3 in 1 Zigbee Remote
- `ROB_200-017-1` ROBB Zigbee smart plug
- `RS 227 T` Innr GU10 spot 420 lm, dimmable, white spectrum
- `S520567` Schneider Electric Roller shutter
- `SLT3C` Hive Heating thermostat remote control
- `SR-ZGP2801K2-DIM` Sunricher Pushbutton transmitter module
- `SR-ZGP2801K4-DIM` Sunricher Pushbutton transmitter module
- `SWITCH-ZR02` eWeLink Zigbee smart switch
- `THS317-ET` OWON Temperature and humidity sensor
- `TS1101_dimmer_module_1ch` TuYa Zigbee dimmer module 1 channel
- `WB-MSW-ZIGBEE v.3` Sprut.device Wall-mounted Zigbee sensor
- `WK34-EU` LELLKI Power socket EU (with power monitoring)
- `WP30-EU` LELLKI Power cord 4 sockets EU (with power monitoring)
- `WP34-EU` LELLKI Outlet power cord with 4 sockets and 2 USB
- `WSD500A` TuYa Temperature & humidity sensor
- `XF-EU-S100-1-M` LELLKI Touch switch 1 gang (with power monitoring)
- `ZFU-1D-CH` Siglis zigfred uno smart in-wall switch
- `ZM-RM02` Zemismart Smart 6 key scene switch

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.22.2'>1.22.2</a>.</em>

**Comment by c0mplex1** [comment]
Great. 👍 
Thank you.

**Comment by Volcano-28** [comment]
Unfortunately my new support request https://github.com/Koenkk/zigbee2mqtt/issues/10300 wasn't solved or even commented.

**Comment by flaektrem** [comment]
@Koenkk Is it safe to update firmware on CC2652 (ZigStar LAN Gateway) without loosing any paired devices? I can´t find any instructions for updating fw, just flash new fw.

**Reply by Koenkk** [reply]
It is safe, updating is the same procedure as reflashing the fw (https://www.zigbee2mqtt.io/guide/faq/#what-does-and-does-not-require-repairing-of-all-devices)

**Reply by flaektrem** [reply]
Ahh, missed that little section in the docs. Thank you

**Reply by kevincaradant** [reply]
@Koenkk 
What is the benefits to have the latest fw ? I mean you said: "allow to set the setting transmit power".
Where is it possible to set it ? The UI of Z2M ? 

Also, what is the current setting transmit power on the "previous" firmware ? Is it 9 like this one ? 

Last question, Is it better to put 5 or 20 ? More it's high, better will be the link quality, but I guess, we will have more  interference with the neighbour wifi for example or I totally wrong on that ? 

Thank you 

Edit: I can answer my self about the power transmit default value in the prvious one.
 "description": "Transmit power of adapter, only available for Z-Stack (CC253*/CC2652/CC1352) adapters, CC2652 = 5dbm, CC1352 max is = 20dbm (5dbm default)",
  So it's 5.
  
   Why only 5 ? I thought it was 20 when we talk about it the last year :)

**Reply by Koenkk** [reply]
Because more != better, the default will be 9dbm for CC1352P and CC2652P coordinators.

**Reply by kevincaradant** [reply]
Ok thank you,  but I'm not familiar with this , that's why I asked if it was useful to upgrade it with a CC1352P like mine or not really :)

**Reply by segaura** [reply]
> safe, updating is the same procedure as reflashing the fw

Really cool.
Is there a way, maybe in some experimental branch, to move from Conbee II to a CC2652x-based coordinator without re-pairing?
I'll be glad to offer my objects and time for testing.

p.s. thank you also for SWITCH-ZR02 support!!

**Comment by jeroenvdd** [comment]
hi after flashing my slae.sh stick and updateing, most things seem to work, but my ikea motion sensors are having troubles,
![image](https://user-images.githubusercontent.com/6975276/148256538-1e3c046a-036d-4db0-a213-4d3672db555a.png)

any clue on how to get these back up?

**Reply by kevincaradant** [reply]
Did you tried to pair it again or reconfigure it again (from the UI) ? What is the issue ? the Last Seen ?

**Reply by jeroenvdd** [reply]
> Did you tried to pair it again or reconfigure it again (from the UI) ? What is the issue ? the Last Seen ?

if i pres reconfigure in UI it times out, and problem with repairing is i have lots of them :D and you need to pull em of the wall and physically push the button on the back..

**Reply by kevincaradant** [reply]
OK I see :). 

I believe that I read somewhere that the power transit can create some issues with some devices. Maybe the new value at 9DB is the reason ? I didn't update yet. I didn't have my answer about the power transit editing ( question just above than your ^^ ) but I guess you can set it in the UI ? Try 5 Db like the previous firmware maybe ? Otherwise I don't know sorry 

Edit; in the doc of Koenkk I saw this:

```
Updating the coordinator firmware.
If after flashing you fail to control your devices it helps to:
Wait a few minutes so that the Zigbee network can settle.
Send Zigbee commands (e.g. turn on/off) to the device.
Reboot the device (unplug and plug power).
```

**Comment by Safs** [comment]
Is time synchronisation fixed for BHT-002-GCLZB?

**Reply by samourai64** [reply]
Yes good idea. I have the same problem and the program mode is not possible because time is never good
Thanks

**Reply by nicoxygen** [reply]
same problem too

**Comment by Safs** [comment]
With 1.22.1-1  version synchronisation working well.

**Reply by nicoxygen** [reply]
with 1.22.2 syncronize time on BHT-002 don't work
